egs/mgb2/asr1/run.sh

```diff
@@ -0,0 +1,322 @@
+#!/bin/bash
+# E2E_mgb2
+# Copyright (C) 2020 Kanari AI  (Amir Hussein)
+
+
+# To use this recipe you'll need
+# 1) Download MGB-2 corpus from https://arabicspeech.org/mgb2 into a 
+#    DB directory. After downloading, it must have DB/{train,dev,test}.tar.gz
+
+
+. ./path.sh || exit 1;
+. ./cmd.sh || exit 1;  
+
+# general configuration
+backend=pytorch
+stage=1   # start from -1 if you need to start from data download
+stop_stage=100
+ngpu=1         # number of gpus ("0" uses cpu, otherwise use gpu)
+debugmode=1
+dumpdir=dump   # directory to dump full features
+N=0            # number of minibatches to be used (mainly for debugging). "0" uses all minibatches.
+verbose=0      # verbose option
+resume=      # Resume the training from snapshot
+subset=1100  # in case we want to use subset of training data
```

Comment:
should we do this number as default?

Reply:
yes, this is the subset that we used during hyperparameter tuning

Reply:
@sw005320 trust all is well.
I have the model already. i can upload it to your drive.